x-pack/plugins/observability/public/components/app/resources/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiListGroup, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+const resources = [
+  {
+    iconType: 'documents',
+    label: i18n.translate('xpack.observability.resources.documentation', {
+      defaultMessage: 'Documentation',
+    }),
+    // TODO: caue what's the url?
+    href: 'https://www.elastic.co',
+  },
+  {
+    iconType: 'editorComment',
+    label: i18n.translate('xpack.observability.resources.forum', {
+      defaultMessage: 'Discuss forum',
+    }),
+    // TODO: caue what's the url?
+    href: 'https://www.elastic.co',
+  },
+  {
+    iconType: 'training',
+    label: i18n.translate('xpack.observability.resources.training', {
+      defaultMessage: 'Training and webinars',
+    }),
+    // TODO: caue what's the url?
+    href: 'https://www.elastic.co',
```

Comment:
@cyrille-leclerc Can you check which of the following options we want to go with?

We could link to https://www.elastic.co/training/observability-fundamentals which is a free online course for learning the basics of Observability.

Otherwise there's a https://www.elastic.co/training/ or https://www.elastic.co/videos/ which offers other resources. Some are paid, some are free to sign up. Not sure of a our strategy here. I think the Observability fundamentals sounds intriguing for this kind of page and more directly relevant than the rest of the content (which can be explored after the fact). I'd change the title of the link to "Observability fundamentals" but keep the icon.

Reply:
The observability fundamental training is a free 2 hours training: https://www.elastic.co/training/observability-fundamentals
We have a more comprehensive (paying) training (32h) that you can find here: https://www.elastic.co/training/elastic-observability-engineer 
We have the Observability certification (not released yet): https://www.elastic.co/training/elastic-certified-observability-engineer

We are soon going to release a bunch of "Quick Start" (APM, Logging and Metrics) that will be hosted directly on our cloud portal. I think we should have them here on the homepage, they are key in term of customer enablement and to drive our cloud strategy.

Reply:
Thanks @melvynator